### PR TITLE
executor/inspect: fix tikv threshold cpu check diagnose rule

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve?

Fix tikv thread cpu threshold check. Different TiKV instance may has different config, so we should use the corresponding config value to check the threshold.

### What is changed and how it works?

Change the threshold cpu check  diagnose rule by change sql.

### Check List 

Tests 

- Unit test
- Manual test (add detailed scripts or steps below)

### CODE REVIEW

**Comment by reafans** [comment]
lgtm

File: executor/inspection_result.go [hunk 1]
```diff
@@ -630,10 +630,11 @@ func (thresholdCheckInspection) inspectThreshold1(ctx context.Context, sctx sess
 
 		var sql string
 		if len(rule.configKey) > 0 {
-			sql = fmt.Sprintf("select t1.instance, t1.cpu, t2.threshold, t2.value from "+
-				"(select instance, max(value) as cpu from metrics_schema.tikv_thread_cpu %[4]s and name like '%[1]s' group by instance) as t1,"+
-				"(select value * %[2]f as threshold, value from information_schema.cluster_config where type='tikv' and `key` = '%[3]s' limit 1) as t2 "+
-				"where t1.cpu > t2.threshold;", rule.component, rule.threshold, rule.configKey, condition)
+			sql = fmt.Sprintf("select t2.instance, t1.cpu, (t2.value * %[2]f) as threshold, t2.value from "+
+				"(select instance as status_address, max(value) as cpu from metrics_schema.tikv_thread_cpu %[4]s and name like '%[1]s' group by instance) as t1 join "+
```
Comment (lonng) [inline executor/inspection_result.go]: The PR ` https://github.com/pingcap/tidb/pull/15548` tries to convert the status address to instance address. The `instance as status_address` will be improper.

Comment (crazycs520) [reply executor/inspection_result.go]: Great, after https://github.com/pingcap/tidb/pull/15548 merge, i will change this .

File: executor/inspection_result_test.go [hunk 2]
```diff
@@ -181,13 +188,38 @@ func (s *inspectionResultSuite) TestThresholdCheckInspection(c *C) {
...
+	fpName := "github.com/pingcap/tidb/executor/mockMergeMockInspectionTables"
+	c.Assert(failpoint.Enable(fpName, "return"), IsNil)
+
+	// Mock for metric table data.
+	fpName2 := "github.com/pingcap/tidb/executor/mockMetricsTableData"
+	c.Assert(failpoint.Enable(fpName2, "return"), IsNil)
+
+	ctx := context.WithValue(context.Background(), "__mockInspectionTables", configurations)
+	ctx = context.WithValue(ctx, "__mockMetricsTableData", mockData)
+	ctx = failpoint.WithHook(ctx, func(_ context.Context, fpname2 string) bool {
```
Comment (Deardrops) [inline executor/inspection_result_test.go]:
```suggestion
	ctx = failpoint.WithHook(ctx, func(_ context.Context, currName string) bool {
```

**Pull request merged**